R/Model-class.R

```diff
@@ -205,6 +207,17 @@ LogisticNormal <- function(mean, cov, ref_dose = 1) {
   )
 }
 
+## default constructor ----
+
+#' @rdname LogisticNormal-class
+#' @examples
+#' .DefaultLogisticNormal()
```

Comment:
because users won't use it, I would not put an example for these default constructors at all. Instead, you could add a small paragraph in one of the vignettes to explain this, give 1-2 examples, and highlight that it is for developers

Reply:
If we omit the `@example` for the `.DefaultXXX()` constructors, then they won't appear in the Usage section of the online doc for each class constructor.  If that is the case, then the statement that end users typically won't  use them would be out-of-context.  Is that sensible?

I am happy to adapt one of the vignettes. Perhaps simulations or migration?

Reply:
Yes, I think it would be better that they don't appear in the usage section of the online docs. Because they are not intended for users, but just for developers. Then we can modify the mentioned statement or remove it.
Yes we could add that few sentences and examples then to the migration vignette, sounds good!

Reply:
As discussed: leave as is.  No action.